src/hotspot/share/oops/arrayOop.hpp

```diff
@@ -83,9 +73,30 @@ class arrayOopDesc : public oopDesc {
                                sizeof(arrayOopDesc);
   }
 
+  // Header size computation.
```

Comment:
Any reasons why this is moved? Visibility change? Would it be uglier to do in place?